manifests/manifests-servicemesh/1.1.0/servicemeshoperator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,404 @@
+          - delete
+          - get
+          - list
+          - patch
+          - watch
+        - apiGroups:
+          - security.openshift.io
+          resources:
+          - securitycontextconstraints
+          resourceNames:
+          - privileged
+          verbs:
+          - use
+        - apiGroups:
+          - ''
+          resources:
+          - nodes
+          - nodes/proxy
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - authentication.k8s.io
+          resources:
+          - tokenreviews
+          verbs:
+          - create
+        - nonResourceURLs:
+          - /metrics
+          verbs:
+          - get
+      deployments:
+      - name: istio-operator
+        spec:
+
+          replicas: 1
+          selector:
+            matchLabels:
+              name: istio-operator
+          template:
+            metadata:
+              labels:
+                name: istio-operator
+            spec:
+              serviceAccountName: istio-operator
+              containers:
+              - name: istio-operator
+                image: registry.redhat.io/openshift-service-mesh/istio-rhel8-operator:1.1.0
+                ports:
+                - containerPort: 11999
+                  name: validation
+                - containerPort: 60000
+                  name: metrics
+                command:
+                - istio-operator
+                imagePullPolicy: Always
+                env:
+                - name: WATCH_NAMESPACE
+                  value: ''
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: istio-operator
+                - name: ISTIO_CNI_IMAGE_V1_0
+                  value: registry.redhat.io/openshift-service-mesh/istio-cni-rhel8:1.0.4
+                - name: ISTIO_CNI_IMAGE_V1_1
+                  value: registry.redhat.io/openshift-service-mesh/istio-cni-rhel8:1.1.0
+  customresourcedefinitions:
+    required:
+    - name: kialis.kiali.io
+      version: v1alpha1
+      kind: Kiali
+      displayName: Kiali
+      description:  A configuration file for a Kiali installation.
+    - name: jaegers.jaegertracing.io
+      version: v1
+      kind: Jaeger
+      displayName: Jaeger
+      description: A configuration file for a Jaeger installation.  
+    owned:
+    - name: servicemeshmemberrolls.maistra.io
+      version: v1
+      kind: ServiceMeshMemberRoll
+      displayName: Istio Service Mesh Member Roll
+      description: A list of namespaces in Service Mesh
+    - name: servicemeshmembers.maistra.io
+      version: v1
+      kind: ServiceMeshMember
+      displayName: Istio Service Mesh Member
+      description: Marks the containing namespace as a member of the referenced Service Mesh
+      specDescriptors:
+      - JSONPath: .spec.controlPlaneRef.namespace
```

Comment:
Should `Control Plane Namespace` and `Control Plane` have displayName fields?

Reply:
We're just pulling this from maistra/istio-operator, which is where it should be fixed (i.e. the csv files are still being maintained in istio-operator).

Reply:
Ok. Thanks. Created https://issues.redhat.com/browse/MAISTRA-1232 to track it.